plugins/jobs/kafka/jobs_kafka_test.go

```diff
@@ -797,6 +797,74 @@ func TestKafkaOTEL(t *testing.T) {
 	})
 }
 
+func TestKafkaPingFailed(t *testing.T) {
+	cont := endure.New(slog.LevelError)
+
+	cfg := &config.Plugin{
+		Version: "v2023.1.0",
+		Path:    "configs/.rr-kafka-ping-failed.yaml",
+		Prefix:  "rr",
+	}
+
+	l, _ := mocklogger.ZapTestLogger(zap.DebugLevel)
+	err := cont.RegisterAll(
+		cfg,
+		&server.Plugin{},
+		&rpcPlugin.Plugin{},
+		&jobs.Plugin{},
+		&kp.Plugin{},
+		&otel.Plugin{},
+		l,
+		&resetter.Plugin{},
+		&informer.Plugin{},
+	)
+	assert.NoError(t, err)
+
+	err = cont.Init()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = cont.Serve()
+
+	assert.NotNil(t, err, "Failed: server should not run")
+	assert.Contains(t, err.Error(), "ping kafka was failed: unable to dial: dial tcp 127.0.0.1:9093: connect: connection refused")
+}
+
+func TestKafkaPingOk(t *testing.T) {
+	cont := endure.New(slog.LevelError)
+
+	cfg := &config.Plugin{
+		Version: "v2023.1.0",
+		Path:    "configs/.rr-kafka-ping-ok.yaml",
+		Prefix:  "rr",
+	}
+
+	l, oLogger := mocklogger.ZapTestLogger(zap.DebugLevel)
+	err := cont.RegisterAll(
+		cfg,
+		&server.Plugin{},
+		&rpcPlugin.Plugin{},
+		&jobs.Plugin{},
+		&kp.Plugin{},
+		&otel.Plugin{},
+		l,
+		&resetter.Plugin{},
+		&informer.Plugin{},
+	)
+	assert.NoError(t, err)
+
+	err = cont.Init()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = cont.Serve()
```

Comment:
After `Serve` container should be stopped. Logs should be checked after stop. Like here for example: [link](https://github.com/roadrunner-server/rr-e2e-tests/blob/master/plugins/jobs/kafka/jobs_kafka_test.go#L253). GitHub Actions are a bit slow, so there is a chance that after `Serve` you won't see a `ping kafka: ok` immediately.

```go
_, err = cont.Serve()
time.Sleep(time.Second) // at least, since `Serve` is async operation internally
err = cont.Stop()
assert.NoError(t, err)
// check for the message here.